### PR TITLE
Update language selector on cms pages to only display available languages

### PR DESCRIPTION
**Pull request by Kenneth-Siu**
****

### CODE REVIEW

File: core/middleware.py [hunk 1]
```diff
@@ -19,15 +15,5 @@ def process_request(self, request):
                     ["https"] + list(urlsplit(request.get_raw_uri())[1:])))
 
 
-class LocaleQuerystringMiddleware(LocaleMiddleware):
-
-    def process_request(self, request):
-        super().process_request(request)
-        language_code = helpers.get_language_from_querystring(request)
```
Comment (richtier) [inline core/middleware.py]: FAS's `get_language_from_querystring` supports both `language` and `lang`:

`request.GET.get('language') or request.GET.get('lang')`

if we made this change then urls with `?language=foo` would stop working how we want them to

as a workaround for that, we can add some middleware that redirects `?language=foo` to `?lang=foo` (or whatever the param that `directory_components.middleware.LocaleQuerystringMiddleware',` expects.

something as gooey as 

```
       if 'language' in self.request.GET:
           url = self.request.get_full_path()
           return redirect(url.replace('language=', 'lang='))
```
would do the job

Comment (Kenneth-Siu) [reply core/middleware.py]: It originally only supported lang.

While I was doing the language switcher on the various repositories, I tried to make sure everything would work fine by accepting both language and lang. This was never released.

And then Edie switched it back to lang everywhere (kind of. You know my thoughts on it from the invest language swithcer update.)

So we're just using lang and it's fine not to accept language.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,15 +9,17 @@
...
 - Upgraded [Constants][directory-constants].
 - Removed application-level IP filtering (replaced with routing solution at platform level).
 - Added Investment Support Directory profile.
 - Added fallback cache for API
-- GA360 changes FAB search to use q instead of term. Redirect also inplace for backwards compatibility. 
+- GA360 changes FAB search to use q instead of term. Redirect also in place for backwards compatibility.
 
 **Fixed bugs:**
 - Upgraded urllib3 to fix [vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2019-11324)
+- Fixed 500 internal server error response on contact forms caused by slugs
```
Comment (richtier) [inline CHANGELOG.md]: this bug has not reached prod/stage, so no need to include it in the release notes

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,15 +9,17 @@
 - Upgraded [API client][directory-api-client], [Forms client][directory-forms-api-client] and because [CMS client][directory-cms-client] upgrade results in [Client core][directory-client-core] being upgraded.
 - Added `DIRECTORY_CLIENT_CORE_CACHE_EXPIRE_SECONDS` env var.
 - Use international header and base templates provided by [Directory components][directory-components].
+  - Implement language switching introduced in the international header and footer
```
Comment (richtier) [inline CHANGELOG.md]: funky indentation :)

**Pull request merged**